### ISSUE TITLE
[plugin-site-issues] error `github.appId is required`  - unable to be "helmfile-diffed" or "helmfile-applied" 

### ISSUE THREAD

**Issue by dduportal** [jenkins-infra/kubernetes-management#1862]
In the contedt of #1861 , getting the following error:

```text
in clusters/prodpublick8s.yaml: command "/usr/local/bin/helm" exited with non-zero status:

PATH:
  /usr/local/bin/helm

ARGS:
  0: helm (4 bytes)
  1: diff (4 bytes)
  2: upgrade (7 bytes)
  3: --reset-values (14 bytes)
  4: --allow-unreleased (18 bytes)
  5: plugin-site-issues (18 bytes)
  6: jenkins-infra/plugin-site-issues (32 bytes)
  7: --version (9 bytes)
  8: 0.1.2 (5 bytes)
  9: --namespace (11 bytes)
  10: plugin-site (11 bytes)
  11: --values (8 bytes)
  12: /tmp/helmfile388112415/plugin-site-plugin-site-issues-values-7d76cd8b4 (70 bytes)
  13: --values (8 bytes)
  14: /tmp/helmfile089001385/plugin-site-plugin-site-issues-values-c64cb44f7 (70 bytes)
  15: --suppress-secrets (18 bytes)
  16: --no-color (10 bytes)
  17: --context (9 bytes)
  18: 2 (1 bytes)

ERROR:
  exit status 1

EXIT STATUS
  1

STDERR:
  Error: Failed to render chart: exit status 1: Error: execution error at (plugin-site-issues/templates/secret.yaml:13:20): github.appId is required
  Use --debug flag to render out invalid YAML
  Error: plugin "diff" exited with error

COMBINED OUTPUT:
  Error: Failed to render chart: exit status 1: Error: execution error at (plugin-site-issues/templates/secret.yaml:13:20): github.appId is required
  Use --debug flag to render out invalid YAML
  Error: plugin "diff" exited with error
```

**Comment by dduportal** [comment]
Found the issue: the YAML keys in the secrets weren't correct.

TL;DR;

As per https://github.com/jenkins-infra/helm-charts/blob/2e9f956a794ec7d98072688f8fb6db8c9e5b80c8/charts/plugin-site-issues/values.yaml#L90-L93

```yaml
github:
  appId: <REDACTED>
  appPrivateKey: <REDACTED>
```

in the SOPS's secrets file, instead of 

```yaml
github:
  app_id: <REDACTED>
  app_private_key: <REDACTED>
```
